openmls/tests/book_code.rs

```diff
@@ -324,12 +324,6 @@ fn book_operations(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvide
     if let ProcessedMessageContent::StagedCommitMessage(staged_commit) =
         alice_processed_message.into_content()
     {
-        let update = staged_commit
-            .commit_update_key_package()
-            .expect("Expected a KeyPackage.");
-        // Check that Bob updated
-        assert_eq!(update.credential(), &bob_credential.credential);
-
```

Comment:
I removed these checks, because there is no more `commit_update_key_package` in the staged commit. We could pull it out of the `StagedCommit`, but we'd have to pierce a number of layers to pull something out (`StagedCommit` contains `MemberStagedCommmitState` contains `StagedPublicGroupDiff` and none of those is meant to be accessed outside of tests). I can create all of those getters, but I thought it's probably not worth it for these extra checks.